common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -647,6 +647,9 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
       "Provide the maximum number of partitions of a table that will be batched together during  \n"
         + "repl load. All the partitions in a batch will make a single metastore call to update the metadata. \n"
         + "The data for these partitions will be copied before copying the metadata batch. "),
+    REPL_PARALLEL_COPY_TASKS("hive.repl.parallel.copy.tasks",1000,
```

Comment:
This is kept configurable so that customer can set it according to their needs. It will create the threads only if there are that many files to copy in parallel. And its using the common pool in the method which is shutdown immediately after the task is complete. Even if multiple threads are created, those are just worker threads which will shutdown after the tasks are done. Customer has to set this config based on their memory allocations.

Reply:
Plus across concurrent policies we can't have a common config. This has to be at the policy level. We can reduce the size but I don't think we can track it across concurrent policies.

Reply:
Setting this to 100